main.go

```diff
@@ -50,6 +50,7 @@ func main() {
 	webhook := flag.Bool("webhook", false, "Runs the webhook webserver.")
 	audit := flag.Bool("audit", false, "Runs a one-time audit.")
 	auditPath := flag.String("audit-path", "", "If specified, audits one or more YAML files instead of a cluster")
+	setExitCode := flag.Bool("set-exit-code-on-error", false, "set an exit code of 2 when the audit contains error-level issues.")
```

Comment:
Heads up - this doc doesn't match the `os.Exit(3)` used below. Also, any consideration on just using `1` as the exit code?

Reply:
Will fix.

We decided against `1` so that we could differentiate this expected failure from unexpected failures.